src/test/regress/expected/isolation_dump_global_wait_edges.out

```diff
@@ -29,11 +29,11 @@ step detector-dump-wait-edges:
 
 waiting_transaction_numblocking_transaction_numblocking_transaction_waiting
 
-290            289            f              
+289            288            f              
```

Comment:
Caused by the change in master_drop_sequences, previously a DROP SEQUENCE during CREATE EXTENSION set off a distributed transaction.